### ISSUE TITLE
Changing the value in one power-select box , it alternately changes the same value for another power-select box

### ISSUE THREAD

**Issue by rinoldsimon** [cibernox/ember-power-select#533]
I used custom-search-action in #ember-power-select, changing the value in one power-select box , it alternately changes the value(sets the same value as selected in 1st select box)for another power-select box too.

Below is the handlebar and js code:

    {{#each model.hpqualifications as |hpqualification|}}
    {{#power-select
      selected=hpqualification.hoprofile
      search=(action "hoProfile")
      onchange=(action (mut hpqualification.hoprofile.name) value="name")
      as |repo|
    }}
      {{repo.name}}
    {{/power-select}}
    {{/each}}

<br>

    import Ember from 'ember';
    export default Ember.Component.extend({
    actions: {
     hoProfile(term){
       if (Ember.isBlank(term)) { return []; }
       const url = `//localhost:3000/betaweb/filters/hoprofiles?  name=${term}`;
       return Ember.$.ajax({ url }).then(json => json.hoprofiles);
     }
    }
    });
The above code generated two select box since #each method has two data.

Everything works fine. But when selecting the value from dropdown for 1st select box, the 2nd select box too changes to the same value. 

How to fix this ?

Please refer to my [stackoverflow][1] question for other files and data format. 

  [1]: http://stackoverflow.com/questions/37645164/ember-power-select-how-to-set-a-preselected-value-in-ember-power-select-custom

**Comment by cibernox** [comment]
Is the answer in stackoverflow solved? I tried to understand the what was going on but the only problem I spotted is the same the other user point you to.

**Comment by rinoldsimon** [comment]
yeah the answer got solved. I modified my js in the following way:
```
hoProfile(term){
  if (Ember.isBlank(term)) { return []; }
    return this.get('store').query('hoprofile', {name: term});
}
```
The problem is in my search action. Because it is returning regular json objects, and when assigning them with the (mut) action, it wont really change the reference, it will just rename existing object.
So i tried querying the data using "store.query" method. 

Finally it got worked !
